code/src/java/pcgen/core/SettingsHandler.java

```diff
@@ -2276,19 +2116,6 @@ private static void readFilePaths()
 			in = new FileInputStream(FILE_LOCATION);
 			getFilepathProp().load(in);
 			String fType = SettingsHandler.getFilePaths();
-
-			if ((fType == null) || (fType.length() < 1))
```

Comment:
I'm unsure how this is a safe removal?

Reply:
I need to go further. Its entirely unused.